moto/glue/responses.py

```diff
@@ -328,7 +328,7 @@ def filter_crawlers_by_tags(self, crawlers, tags):
         return [
             crawler.get_name()
             for crawler in crawlers
-            if self.is_tags_match(crawler.tags, tags)
+            if self.is_tags_match(self.glue_backend.get_tags(crawler.arn), tags)
```

Comment:
```suggestion
            if self.is_tags_match(crawler.arn, tags)
```
This would be easier to read. It does mean we have to move the `get_tags()`-call to inside the `is_tags_match`-method, and remove the duplication in `filter_jobs_by_tags`

Reply:
Agreed. I've moved that get_tags() call into the is_tags_match method.